backend/server/routers/planner.py

```diff
@@ -76,3 +76,8 @@ async def validate_term_planner(plannerData: PlannerData):
         user.year += 1
 
     return {"courses_state": coursesState}
+
+@router.post("/saveLocalStorage/")
+def save_local_storage(localStorage: LocalStorage):
+    #TODO: replace this with a real implementation
+    print(localStorage)
```

Comment:
i think this should also take in a token.

Reply:
if we take a token, we can make an actual implementation of this. I have the database set up, so you can add to it by using a token as a key, with an objectId is a value for that token to be mapped to. look at databases.py.